docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: '3.7'
+services:
+
+  ciDB:
+    image: aqts_capture_db_ci
+    build:
+      context: .
+      dockerfile: ./database/ci/Dockerfile
+    networks:
+      aqts_capture:
+        ipv4_address: ${DB_CI_IPV4}
+    environment:
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - AQTS_DATABASE_NAME=${AQTS_DATABASE_NAME}
+      - AQTS_DB_OWNER_USERNAME=${AQTS_DB_OWNER_USERNAME}
+      - AQTS_DB_OWNER_PASSWORD=${AQTS_DB_OWNER_PASSWORD}
+      - AQTS_SCHEMA_NAME=${AQTS_SCHEMA_NAME}
+      - AQTS_SCHEMA_OWNER_USERNAME=${AQTS_SCHEMA_OWNER_USERNAME}
+      - AQTS_SCHEMA_OWNER_PASSWORD=${AQTS_SCHEMA_OWNER_PASSWORD}
+      - AQTS_DATABASE_ADDRESS=127.0.0.1
+    ports:
+      - ${DB_CI_PORT}:5432
+    container_name: ${AQTS_DATABASE_ADDRESS}_ci
+
+  db:
+    image: aqts_capture_db
+    build:
+      context: .
+    networks:
+      aqts_capture:
+        ipv4_address: ${DB_IPV4}
+    environment:
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+    ports:
+      - ${DB_PORT}:5432
+    container_name: ${AQTS_DATABASE_ADDRESS}
```

Comment:
Is the container_name intended to be the db address here?

Reply:
yes - you could argue the parameter is serving two purposes - a server name when using the docker network (like the travis build) or an IP address when not (some local runs) 
I think it is less confusing to have them be same.